app/javascript/components/supplemental_application/sections/Lease.js

```diff
@@ -217,6 +217,7 @@ const Lease = ({ form, values, store }) => {
           <FormGrid.Item>
             <SelectField
               label='Preference Used'
+              selectValue={values.lease.preference_used || NONE_PREFERENCE_LABEL}
```

Comment:
This is a little weird, the handleCancelClicked handler wasn't setting this value back to none properly

to repro:
1. start with appication with lease that has "none" preference
2. change preference to something other than none
3. Click cancel
4. Note that the preference field hasn't reverted to "None" like it should have.


The ` || NONE_PREFERENCE_LABEL` here makes it so that when preference_used is undefined it sets the value to "None", since for some reason undefined doesn't revert to the "None" option